src/main/java/net/sf/mzmine/util/scans/ScanUtils.java

```diff
@@ -621,7 +621,7 @@ public static int findBestFragmentScan(RawDataFile dataFile, Range<Double> rtRan
    * Sorted list (best first) of all MS2 fragmentation scans with n signals >= noiseLevel in the
    * specified or first massList, if none was specified
    * 
-   * @param row
+   * @param row all MS2 scans of all features in this row
```

Comment:
ScanUtils

getMassListOrFirst to always work on masslists (the user can supply a masslist name or it uses the first masslist)

listAllFragmentScans(row, sortingMode)